tests/test_dojo.py

```diff
@@ -74,7 +79,102 @@ def test_raise_error_on_duplicate_persons(self):
 		self.dojo.create_room("Hunter", "Office")
 		self.dojo.add_person("staff", "John", "Ngravo")
 		self.assertEqual(self.dojo.add_person("staff", "John", "Ngravo"), "Person already exists")
-			 
+
+class TesstPrintFunctions(unittest.TestCase):
+	"""A class to test the print functionalities in v1"""
+
+	def setUp(self):
+		self.dojo = Dojo()
+
+	def test_print_names_of_people_in_room(self):
+		#create rooms and add persons
+		self.dojo.create_room("Nairobi", "Office")
+		self.dojo.add_person("fellow", "ernest", "achesa", "N")
+		self.dojo.add_person("staff", "george", "wanjala")
+		#snapshot to store sys.stdout
+		temp = sys.stdout
+		#capture will hold the output being printed
+		capture = StringIO()
+		sys.stdout = capture
+		#call functions
+		self.dojo.print_room("Nairobi")
+		#end functions
+		#revert to snapshot of sys.stdout
+		sys.stdout = temp
+		#the printed value that was captured
+		output = capture.getvalue()
+		heading = "\x1b[33m\nROOM NAME: NAIROBI\tROLE\n"
+		expected_output = heading + "\x1b[0m\nErnest Achesa \t\tFELLOW\nGeorge Wanjala \t\tSTAFF\n"
+		self.assertEqual(output, expected_output)
```

Comment:
great idea to capture what should be printed on the screen, good work on comments. 
I don't see where you are testing for an empty room and room that doesn't exist